modules/farm_fd2_school/src/entrypoints/Vue4/App.vue

```diff
@@ -131,14 +132,30 @@ export default {
       pickedPlant: -1,
       cropList: ['ARUGULA', 'ASPARAGUS', 'BEAN', 'RADISH'],
       plantList: [
-        { id: 1, date: '04/02/2019', location: 'D', bed: '' },
-        { id: 2, date: '04/02/2019', location: 'GHANA', bed: 'GHANA-2' },
-        { id: 3, date: '04/02/2019', location: 'GHANA', bed: 'GHANA-4' },
-        { id: 4, date: '06/05/2019', location: 'GHANA', bed: 'GHANA-4' },
+        { id: 1, date: '08/19/2019', location: 'D', bed: '' },
+        { id: 2, date: '09/20/2007', location: 'GHANA', bed: 'GHANA-2' },
+        { id: 3, date: '06/05/2019', location: 'GHANA', bed: 'GHANA-4' },
+        { id: 4, date: '04/02/2019', location: 'GHANA', bed: 'GHANA-4' },
       ],
       unitList: ['BUNCH', 'EACH', 'POUND'],
     };
   },
+  computed: {
+    isButtonDisabled() {
```

Comment:
I think that the name of this computed property be more descriptive of what it does. Maybe `isFormValid` or `submitButtonDisabled` or something similar?